test/conftest.py

```diff
@@ -28,3 +31,46 @@ def drop_all():
     psqlgraph.create_all(pg_graph_driver.engine)
 
     return pg_graph_driver
+
+
+SAMPLES = [
+    dict(
+        bar="bar1",
+        baz="allowed_1",
+        fobble=25,
+        studies=["P1", "P2"],
+        ages=[23, 45],
+        node_id=str(uuid.uuid4())
+    ),
+    dict(
+        bar="bar2",
+        baz="allowed_1",
+        fobble=25,
+        studies=["C1", "P2"],
+        ages=[29, 45],
```

Comment:
Since it only works with text values, what would happen if tried a `props_in` against this numeric `ages` property and numeric values like `[45]`?  Does to raise an exception?  or does it behave oddly?  I'm also curious what happens if we cast the values like `45` to a string. Since that's the right hand side, would it make`?|` work?

Reply:
Using something like `[45]` will raise an invalid sql query exception, using `["45"]` will return an empty result since the string version does not exist in the property values.